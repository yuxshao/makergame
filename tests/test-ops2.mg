object main {
event create
{
  std::printb(true);
  std::printb(false);
  std::printb(true && true);
  std::printb(true && false);
  std::printb(false && true);
  std::printb(false && false);
  std::printb(true || true);
  std::printb(true || false);
  std::printb(false || true);
  std::printb(false || false);
  std::printb(!false);
  std::printb(!true);
  std::print(-10);
  std::print(-(-42));
  std::end_game();
}
}
