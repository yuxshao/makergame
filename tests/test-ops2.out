true
false
true
false
false
false
true
true
true
false
true
false
-10
42
